coderd/cryptokeys/cache.go

```diff
@@ -255,6 +255,7 @@ func (c *cache) cryptoKey(ctx context.Context, sequence int32) (string, []byte,
 
 	keys, err := c.cryptoKeys(ctx)
 	if err != nil {
+		c.mu.Lock() // Re-lock because of defer.
```

Comment:
I think it's clearer to move the Lock call from L262 up like:

```
c.mu.Unlock()
keys, err := c.cryptoKeys(ctx)
c.mu.Lock()
if err != nil {
    return "", nil, xerrors.Errorf("get keys: %w", err)
}
```

That way it's clear that we are unlocking just for the `c.cryptoKeys()` call.